test/parallel/test-os.js

```diff
@@ -182,11 +209,17 @@ assert.ok(pwd.homedir.includes(path.sep));
 assert.strictEqual(pwd.username, pwdBuf.username.toString('utf8'));
 assert.strictEqual(pwd.homedir, pwdBuf.homedir.toString('utf8'));
 
-// Test that the Symbol.toPrimitive functions work correctly
+// Test Symbol.toPrimitive on these functions
 [
+  [+os.freemem, os.freemem()],
```

Comment:
Idea:
replace all the pairs with just the field name and last line to
```js
].forEach((field) => assert.strictEqual(os[field], os[field](), `os.${field} failed Symbol.toPrimitive test`));
```

Reply:
@refack there is sometimes `string` sometimes `number` so the check should be more complicated than just `os[field]`. So I guess it's not worth it and the current assertion is not that bad

Reply:
Since the `*mem` & `uptime` can't be compared with `strictEqual` (the values can change between the two calls), they should be moved to their own loop anyway.

Reply:
I agree with @refack that it would be very good to add the field name as it is difficult to know otherwise from what entry a failure is actually coming from.
This could also be done by either adding a string to each entry you are iterating over as a third entry or by just using the `index` of the `forEach` loop as in `Entry ${i} failed` but I prefer what @refack suggests.

I personally would prefer a third alternative - to just remove the loop altogether as it is unnecessary.
We would even have less lines if we use e.g. `assert.strictEqual(`${os.endianness}`, os.endianness())` directly instead of values from the loop and it would be clear where the error is coming from.

I also do not see any way to compare `freemem` or `uptime` without those being flaky. So please remove those two entries again.